app/src/debug/AndroidManifest.xml

```diff
@@ -86,6 +86,11 @@
             android:label="Trustly Mock Screen"
             android:theme="@style/Hedvig.Theme" />
 
+        <activity
+            android:name=".feature.checkout.CheckoutMockActivity"
+            android:label="Checkout Mock Screen"
+            android:theme="@style/Hedvig.Theme" />
```

Comment:
```suggestion
            android:label="Checkout Mock Screen" />
```

We get the `android:theme` automatically from the `<application>`-declaration in `main/AndroidManifest.xml` - no need to specify it on a per-activity basis 🙂